src/pages/programIndicators/form/ProgramIndicatorFormFields.tsx

```diff
@@ -0,0 +1,277 @@
+import i18n from '@dhis2/d2-i18n'
+import {
+    CheckboxFieldFF,
+    InputFieldFF,
+    SingleSelectField,
+    SingleSelectFieldFF,
+    SingleSelectOption,
+} from '@dhis2/ui'
+import React from 'react'
+import { Field as FieldRFF, Field, useField } from 'react-final-form'
+import {
+    CustomAttributesSection,
+    DefaultIdentifiableFields,
+    DescriptionField,
+    ModelTransferField,
+    SectionedFormSection,
+    SectionedFormSections,
+    StandardFormField,
+    StandardFormSectionDescription,
+    StandardFormSectionTitle,
+} from '../../../components'
+import { ExpressionBuilder } from '../../../components/ExpressionBuilder/ExpressionBuilder'
+import {
+    ModelSingleSelectField,
+    ModelSingleSelectFormField,
+} from '../../../components/metadataFormControls/ModelSingleSelect'
+import { getConstantTranslation, SECTIONS_MAP, useSchema } from '../../../lib'
+import { DisplayableModel } from '../../../types/models'
+import { ColorAndIconField } from '../../dataElements/fields'
+import { AnalyticsPeriodBoundariesField } from './AnalyticsPeriodBoundariesField'
+import { OrgUnitField } from './OrgUnitField'
+
+const section = SECTIONS_MAP.programIndicator
+
+export const ProgramIndicatorsFormFields = () => {
+    const { input: decimalsInput } = useField('decimals')
+    const programFilters = [
+        'id,displayName,programType,programTrackedEntityAttributes[trackedEntityAttribute[id,displayName,valueType]]',
+    ] as const
+
+    const schema = useSchema(SECTIONS_MAP.programIndicator.name)
+    return (
+        <SectionedFormSections>
+            <SectionedFormSection name="basicInformation">
+                <StandardFormSectionTitle>
+                    {i18n.t('Basic information')}
```

Comment:
The initial idea was to try to reuse the `formDescriptor` for these, so that we have one place that defines names and labels.

Eg.

```ts
    const descriptor = useSectionedFormContext<typeof ProgramIndicatorFormDescriptor>()

   ...

           <SectionedFormSection name={descriptor.getSection('basicInformation').name}>
                <StandardFormSectionTitle>
                    {descriptor.getSection('basicInformation').label}
                </StandardFormSectionTitle>
```